### ISSUE TITLE
Sprockets::ArgumentError: Asset logical path has no extension: adapters/application when I try to compile assets

### ISSUE THREAD

**Issue by GCorbel** [emberjs/ember-rails#510]
I just added `ember-rails` to my Gemfile and generated the bootstrap with `rails g ember:bootstrap -g --javascript-engine coffee`. When I try to precompile, I have : 

```
rake aborted!
Sprockets::ArgumentError: Asset logical path has no extension: adapters/application
/var/lib/gems/2.2.0/gems/sprockets-2.12.4/lib/sprockets/asset.rb:36:in `initialize'
/var/lib/gems/2.2.0/gems/sprockets-2.12.4/lib/sprockets/bundled_asset.rb:14:in `initialize'
/var/lib/gems/2.2.0/gems/sprockets-2.12.4/lib/sprockets/base.rb:377:in `new'
/var/lib/gems/2.2.0/gems/sprockets-2.12.4/lib/sprockets/base.rb:377:in `build_asset'
/var/lib/gems/2.2.0/gems/sprockets-2.12.4/lib/sprockets/index.rb:94:in `block in build_asset'
/var/lib/gems/2.2.0/gems/sprockets-2.12.4/lib/sprockets/caching.rb:58:in `cache_asset'
/var/lib/gems/2.2.0/gems/sprockets-2.12.4/lib/sprockets/index.rb:93:in `build_asset'
/var/lib/gems/2.2.0/gems/sprockets-2.12.4/lib/sprockets/base.rb:287:in `find_asset'
/var/lib/gems/2.2.0/gems/sprockets-2.12.4/lib/sprockets/index.rb:61:in `find_asset'
/var/lib/gems/2.2.0/gems/sprockets-2.12.4/lib/sprockets/manifest.rb:211:in `block in find_asset'
/var/lib/gems/2.2.0/gems/sprockets-2.12.4/lib/sprockets/manifest.rb:257:in `benchmark'
/var/lib/gems/2.2.0/gems/sprockets-2.12.4/lib/sprockets/manifest.rb:210:in `find_asset'
/var/lib/gems/2.2.0/gems/sprockets-2.12.4/lib/sprockets/manifest.rb:119:in `block in compile'
/var/lib/gems/2.2.0/gems/sprockets-2.12.4/lib/sprockets/manifest.rb:118:in `each'
/var/lib/gems/2.2.0/gems/sprockets-2.12.4/lib/sprockets/manifest.rb:118:in `compile'
/var/lib/gems/2.2.0/gems/sprockets-rails-2.3.3/lib/sprockets/rails/task.rb:70:in `block (3 levels) in define'
/var/lib/gems/2.2.0/gems/sprockets-2.12.4/lib/rake/sprocketstask.rb:146:in `with_logger'
/var/lib/gems/2.2.0/gems/sprockets-rails-2.3.3/lib/sprockets/rails/task.rb:69:in `block (2 levels) in define'
/var/lib/gems/2.2.0/gems/bugsnag-2.8.12/lib/bugsnag/rake.rb:12:in `execute_with_bugsnag'
Tasks: TOP => assets:precompile
```
This is a Rails 4.2.5.2 application.

**Comment by tricknotes** [comment]
Thanks for your report @GCorbel !
The generator should be fixed to generate `xxx.js.coffee` for Sprockets 2, not a  `xxx.coffee`.

**Comment by tricknotes** [comment]
> The generator should be fixed to generate `xxx.js.coffee` for Sprockets 2, not a  `xxx.coffee`.

This is incorrect.
This is a `ember-es6_template`'s bug that fixed by https://github.com/tricknotes/ember-es6_template/pull/7.

@GCorbel 
I released a new version [0.4.4](https://rubygems.org/gems/ember-es6_template/versions/0.4.4) including fixes for this issue :rocket: 
Could you try it? If not fixed yet, feel free to reopen this issue :smile:
